K2Bridge.Tests.UnitTests/Visitors/HistogramAggregationVisitorTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace UnitTests.K2Bridge.Visitors
+{
+    using global::K2Bridge.Models.Request.Aggregations;
+    using global::K2Bridge.Visitors;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class HistogramAggregationVisitorTests
+    {
+        [TestCase(ExpectedResult = "_data | summarize count() by ['key%False'] = bin(['field'], 20)\n| order by ['key%False'] asc")]
+        public string HistogramVisit_WithSimpleAggregation_ReturnsValidResponse()
+        {
+            var histogramAggregation = new HistogramAggregation()
+            {
+                Field = "field",
+                Interval = 20,
+                MinimumDocumentCount = 0,
+                Key = "key",
+            };
+
+            var visitor = new ElasticSearchDSLVisitor(SchemaRetrieverMock.CreateMockSchemaRetriever());
+            visitor.Visit(histogramAggregation);
+
+            return histogramAggregation.KustoQL;
+        }
+
+        [TestCase(ExpectedResult = "_data | summarize count() by ['key%False'] = bin(['field'], 20)\n| order by ['key%False'] asc")]
```

Comment:
how is the min doc count affect the result?

Reply:
Lost the update while last git rebase :( 
Fixed